src/coreclr/jit/lclvars.cpp

```diff
@@ -7388,11 +7390,6 @@ void Compiler::lvaDumpEntry(unsigned lclNum, FrameLayoutState curState, size_t r
         printf(" HFA(%s) ", varTypeName(varDsc->GetHfaType()));
     }
 
-    if (varDsc->lvLiveInOutOfHndlr)
-    {
-        printf(" EH");
-    }
-
     if (varDsc->lvDoNotEnregister)
```

Comment:
Seems like we might still want to show this?

Reply:
We already show it as `EH-live` below so this one was redundant.

https://github.com/dotnet/runtime/blob/7ab7eaa24c318cf67ff2b5bc9713b61a86c567b4/src/coreclr/jit/lclvars.cpp#L7499-L7502